.github/workflows/zfs-tests-functional.yml

```diff
@@ -61,6 +55,9 @@ jobs:
         sudo rm -rf /opt/ghc
         sudo rm -rf "/usr/local/share/boost"
         sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+        # Flush kernel buffer
+        sudo dmesg -c
```

Comment:
We should get a clean instance for each build so I'm not sure this buys us anything, but no real objection.

Reply:
The reason I added flushing in here is to get smaller and cleaner `dmesg` logs for each test case execution. I felt like it is not that important to have a full dump from second 0 in all log files.

Reply:
`dmesg -c` will print it's buffer into CI's log, large log will make run and log download itself slower. I propose to remove this addition.

Reply:
Sorry, didn't see earlier answer. Looked at action log, it gave ~600 lines, not so bad, but I'd prefer to move it in separate step for logs readability or replace `-c` with `-C`.